daily-patch-information/src/main/java/org/wso2/patchinformation/MainEmailSender.java

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
You need to follow the WSO2 way of structuring the component. For example, the source should go under components directory, and then the component name should be the submodule of that. please check - https://github.com/wso2/carbon-analytics for example. And then add a root pom.xml, etc to enable some one to build the project.